src/dex/api/sundaeswap-v3-api.ts

```diff
@@ -0,0 +1,122 @@
+import { BaseApi } from './base-api';
+import { Asset, Token } from '../models/asset';
+import { LiquidityPool } from '../models/liquidity-pool';
+import axios, { AxiosInstance } from 'axios';
+import { RequestConfig } from '@app/types';
+import { appendSlash } from '@app/utils';
+import { SundaeSwapV3 } from '@dex/sundaeswap-v3';
+
+export class SundaeSwapV3Api extends BaseApi {
+
+    protected readonly api: AxiosInstance;
+    protected readonly dex: SundaeSwapV3;
+
+    constructor(dex: SundaeSwapV3, requestConfig: RequestConfig) {
+        super();
+
+        this.dex = dex;
+        this.api = axios.create({
+            timeout: requestConfig.timeout,
+            baseURL: `${appendSlash(requestConfig.proxyUrl)}https://stats.sundaeswap.finance/graphql`,
```

Comment:
This API won't return anything about sundae v3.

You'll need to use our new API (which supports both v1 and v3) at https://api.sundae.fi/graphql

Reply:
or for preview, https://api.preview.sundae.fi/graphql